content/use-cases/tokenization/stablecoin-issuer.md

```diff
@@ -516,36 +185,22 @@ To submit transactions reliably, follow these guidelines:
 
 For more information, see [Reliable Transaction Submission](reliable-transaction-submission.html).
 
+### List on an XRPL Native DEX
+
+Decentralized exchanges are integral to the decentralized finance ecosystem. Listing your token on well known DEXs operating on XRPL will enhance its visibility and accessibility, thereby attracting more liquidity. Begin by placing sell offers on a suitable DEX, such as [Sologenic](https://sologenic.org/trade).
```

Comment:
The wording here is a bit misleading: there is _one, global DEX_ built into the XRP Ledger. Sologenic and others are just _interfaces_ into the same shared system. (Offers placed on GateHub can be seen and matched on Sologenic, and vice-versa.)

Also, token issuers don't need to do anything to list on the DEX. They can, if they want, set up a trading account to place offers for their token. This guide should probably provide the advice that they use a separate account for trading than they do for issuing.